Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Black_Hive.xml

```diff
@@ -255,33 +255,33 @@
 							<capacities>
 								<li>Cut</li>
 							</capacities>

```

Comment:
In the RW lore, insects were bioengineered to fight Mechanoids, thus their AP is artificially high.